Add invite new members section (#925)
pulumi/pulumi-hugo: themes/default/content/docs/intro/console/organizations.md

you must associate a GitLab identity with your Pulumi account, and also
be a member of that GitLab group.

## Inviting Members to an Organization

Pulumi organization administrators can invite new members to an organization.

To invite a new member:

1. Select the members tab.

> 2. To invite new members using their email addresses, select **Invite members**. After the new member(s) have been invited, they will receive an email with a link to click to accept the invitation.

Reviewer: i think this makes it slightly easier to understand, but i don't feel strongly

```suggestion
2. To invite a new member using their email address, select **Invite members**. After the new member is invited, they will receive an email allowing them to accept the invite.
```

> 3. To invite new members by sharing a link, select **Copy new invite link**. This will copy a link to your clipboard that you can share directly. 

Reviewer: ```suggestion
3. To invite a new member by sharing a link, select **Copy new invite link**. This will copy a link to your clipboard that you can share directly. 
```

New member invitation links never expire and may only be used one time.

> Pulumi organization administrators can monitor current organization members and pending invitations on the Members console page. For pending invitations, administrators can re-send email invitations, copy links for both email and link generated invitations, revoke invitations, and see any errors that pending members encountered when attempting to accept invitations.

Reviewer 2: > see any errors that pending members encountered when attempting to accept invitations.

How do we see this and what's actually seeable? Would it be worth mentioning where/how (since otherwise you're kind of left wondering)?

Reviewer: it gets displayed in the table, last column. but we can def add more details here.

Author: Added some details here in the latest push along with the SSO auto-fill link later in the page.
